public/app/features/migrate-to-cloud/onprem/Page.tsx

```diff
@@ -78,7 +78,15 @@ function useGetLatestSnapshot(sessionUid?: string, page = 1) {
 
   const getSnapshotQueryArgs =
     sessionUid && lastItem?.uid
-      ? { uid: sessionUid, snapshotUid: lastItem.uid, resultLimit: PAGE_SIZE, resultPage: page }
+      ? {
+          uid: sessionUid,
+          snapshotUid: lastItem.uid,
+          resultLimit: PAGE_SIZE,
+          resultPage: page,
+          resultSortColumn: sortParams?.column ? sortParams.column : undefined,
+          resultSortOrder: sortParams?.direction,
+          errorsOnly: showErrors,
```

Comment:
What about https://developers.grafana.com/ui/latest/index.html?path=/story/forms-switch--controlled instead of a checkbox? According to our docs, checkboxes shouldn't have "instant" effect, only when Submitting a form, while a switch is "instant", and aligns better to me in terms of what the underlaying behaviour is.

Reply:
Nice catch, changed [here](https://github.com/grafana/grafana/pull/103061/commits/21fb97e3fa95696876381ea9ae2c255497049011)